app/konnect/dev-portal/customization/index.md

```diff
@@ -56,3 +56,38 @@ Add a custom Dev Portal domain through your organization's {{site.konnect_short_
 
    {:.note}
    > **Note:** DNS propagation can take a few hours. If after a few hours you can't access the Dev Portal from the custom URL, contact your domain registrar.
+
+## Single Sign-On
+{:.badge .enterprise}
+
+The Dev Portal can be configured to use Single Sign-On (SSO) by a {{site.konnect_short_name}} Admin via the Identity settings in the {{site.konnect_short_name}} Admin interface.
+
+1. In {{site.konnect_short_name}}, open {% konnect_icon dev-portal %} **Dev Portal**, then click **Settings**.
+
+2. Open the **Identity** tab.
+
+   {:.note}
+      > **Note:** Copy the callback URL and enter it in your OIDC Provider
+
+3. Enter the full domain, including subdomain and protocol, into the  **Provider URL** field (also known as **Issuer**). e.g. `https://accounts.google.com` for Google IdP
+
+4. Enter the unique identifier provided by the IdP into the **Client ID** field.
+
+5. Enter the secret used to verify ownership of your IdP client into the **Client Secret** field.
+
+### Configuration Requirements
+
+A form of authentication must always be configured. Built-in and SSO can be used individually or in combination. Each state results in a different user experience represented in this table:
+
+| State | SSO registration | SSO sign in | Built-in registration| Built-in sign in | Information|
+| --- | :-----------: |  ----------- | :-----------: | ----------- |
+| **SSO: Enabled**<br>**Built-in: Disabled** | ✅ | ✅ | ❌ | ❌ | - Developer will be able to register and log in with SSO.<br><br>- Developers registered with Built-in will be forced to use SSO.<br><br>- The Built-in authentication window will not be displayed from the Dev portal. 
+| **SSO: Disabled**<br>**Built-in: Enabled** | ❌ | ❌ | ✅  |✅  | - Developers will be able to register and log in with a username and password.| 
+| **SSO: Enabled**<br>**Built-in: Enabled** | ✅  | ✅ | ✅  | ✅  | - Developers will be able to register and log in with either form of authentication.|
+
+
+### OIDC Details
+
+* If a user account associated to a {{site.konnect_short_name}} developer is removed from the IdP, the {{site.konnect_short_name}} developer account will continue to exist. It must be deleted by a {{site.konnect_short_name}} admin from the {{site.konnect_short_name}} dashboard.
```

Comment:
```suggestion
* If a user account associated with a {{site.konnect_short_name}} developer is removed from the IdP, the {{site.konnect_short_name}} developer account is not deleted. A {{site.konnect_short_name}} Admin must delete it from the {{site.konnect_short_name}} dashboard.
```
"...associated with a...." might just be a personal preference, so feel free to ignore that. 

Also, you used "Konnect Admin" before, so I just capped "admin" because I couldn't find a rule in the style guide about it.